src/main/java/com/futurecollars/zad3/Employee.java

```diff
@@ -0,0 +1,25 @@
+package com.futurecollars.zad3;
+
+public class Employee {
```

Comment:
to powinna być klasa abstrakcyjna, dodaj też w niej abstrakcyjną metodę calculateMonthlySalary, tak aby klasy dziedziczącej musiały spełnić kontrakt i ją zaimplementować